lib/ridgepole/dsl_parser/context.rb

```diff
@@ -86,6 +87,11 @@ def add_foreign_key(from_table, to_table, options = {})
 
         raise "Foreign Key `#{from_table}(#{idx})` already defined" if @__definition[from_table][:foreign_keys][idx]
 
+        if @__options[:set_index_to_fk]
+          index_column = options[:column] ||= "#{to_table.singularize}_id"
```

Comment:
`index_column = options[:column] || "#{to_table.singularize}_id"` ?

Reply:
That is certainly true.